### DISCUSSION TITLE
The Mach-O Transition: Darling in the Past 5 Years | Darling Development Blog

### DISCUSSION THREAD

**Discussion by giscus** [darlinghq/darling#1379] [Blog comments]
# The Mach-O Transition: Darling in the Past 5 Years | Darling Development Blog

Darling has been under development for almost five years now, which invites the questions &mdash; what has happened over the past years, are we getting anywhere and when will we get there.

https://blog.darlinghq.org/2017/02/the-mach-o-transition-darling-in-past-5.html

**Comment by facekapow** [comment]
This is a test comment!

**Reply by bugaevc** [reply]
And a test reply \o/

**Reply by renhiyama** [reply]
And this confirms that your tests are successful.

**Comment by bugaevc** [comment]
For anyone else who's confused: look at the date, this post is from 2017. We've just revamped our blog and are testing out comments. A new blog post is coming soon though, stay tuned!

**Comment by CuriousTommy** [comment]
> It is very well possible that building ELFs instead of real PE DLLs bears no disadvantages for Wine; I cannot assess this.

The interesting thing is that [Wine has been going through the process of converting their libraries to a PE format](https://www.winehq.org/announce/5.0#:~:text=***%20PE%20modules%20-%20Most%20modules%20are%20built,are%20copied%20into%20the%20Wine%20prefix%20instead%20of). In a way, it's kind of funny that Wine ended up doing the same thing that we did.
